lib/wp2ghost.js

```diff
@@ -119,10 +119,18 @@ exports.fromStream = function(stream) {
 
       var date;
 
+      date = item['wp:post_date_gmt'];
+      if (date == null) {
+          date = item['wp:post_date'];
+        if (date == null) {
+          date='0000-00-00 00:00:00'
+        }
+      }
+
       if (item['wp:post_date_gmt'] !== "0000-00-00 00:00:00") {
```

Comment:
Adding ` && item['wp:post_date_gmt'] !== undefined` does help in the case `wp:post_date_gmt `is not present but `wp:post_date` is. However if both are missing, as it was in my case the program still crashes silently.

Example of what it crashed on:

```
    <item>
      <title>Ideas, not to be published!</title>
      <link>/blog/58</link>
      <content:encoded><![CDATA[<p>stuff...</p></content:encoded>
      <excerpt:encoded />
      <wp:post_name>58</wp:post_name>
      <wp:post_type>post</wp:post_type>
      <wp:post_id>39</wp:post_id>
      <wp:status>draft</wp:status>
      <pubDate>Fri, 24146140482 15:36:27 +0000</pubDate>
    </item>
```